Project.toml

```diff
@@ -5,8 +5,10 @@ version = "0.3.0"
 
 [deps]
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+RData = "df47a6cb-8c03-5eed-afd8-b6050d6c41da"
```

Comment:
We have a way to add test dependencies via the [extra] section and the [target] section. See this example: https://github.com/JuliaEarth/Variography.jl/blob/f7fd09aa0a705701bf6eb93c52294cc0a13a99ad/Project.toml#L32-L48